app/src/hooks/activeCall.ts

```diff
@@ -68,14 +77,20 @@ export const useActiveCallStatus = (
  * @param activeCall - The active call.
  * @returns - The remote participant identity.
  */
-export const useActiveCallRemoteParticipant = (activeCall: ActiveCall) => {
-  const remoteParticipantId = React.useMemo<string>(() => {
-    if (activeCall?.status !== 'fulfilled') {
-      return '';
-    }
-
-    return activeCall.to || '';
-  }, [activeCall]);
+export const useActiveCallRemoteParticipant = (
+  activeCall: ActiveCall | undefined,
+) => {
```

Comment:
Is the return type here string?

Reply:
Yes, the return type here is just a string.